### ISSUE TITLE
Why?

### ISSUE THREAD

**Issue by Leviterminator** [iblobtouch/iblobtouch.github.io#55]
Why did you stop maintaining the game? You should either add new features, or fix the bugs. But you are doing nothing. And it is 100% sure, that you won't reply, but please note, this issues tab exists, and you should fix the issues the players notice.

**Comment by aryamanarora** [comment]
He doesn't have to do anything... The good thing about GitHub and open source in general is that anyone can help.

**Comment by Leviterminator** [comment]
Yeah, anyone who can code. I am learning it, but i am still a "very beginner", so i can't do all thiese coding stuff.

**Comment by aryamanarora** [comment]
But the great thing about GitHub is that *other people know*!  Even if @iblobtouch isn't maintaining it, we have @Acarii.

I don't know JavaScript either.

**Comment by Leviterminator** [comment]
He is in his team, i guess.

**Comment by DogeisCut** [comment]
hey did you read my sugestions?

**Comment by Leviterminator** [comment]
Of course i have. But what does it have to do with my post?

**Comment by Acarii** [comment]
A lot of my interest in the project has waned in the past few weeks. I'm a newbie programmer, learning exclusively while working on the project and I keep running into issues I can't solve with my current skillset. While the project isn't dead, it's definitely on hiatus for the time being. I simply do not know enough to do what I want to do with the project.

**Edit:** We are open source. Send in code and I can go over if it's a change that should be done or not. Do not be afraid to try and rewrite anything and everything, neither blob or I care how you modify the code we have, so long as the project continues(and you don't take credit you don't deserve.)

**Comment by DogeisCut** [comment]
OH SORRY gee

**Comment by DogeisCut** [comment]
i dont know how to code :( 👍

**Comment by DogeisCut** [comment]
Dang ,don't lose interest, We love this and you, so please continue fixing and adding features. Thank You.
Don't let us down!!!

**Comment by DogeisCut** [comment]
arci

**Comment by DogeisCut** [comment]
acarii please
